eng/Versions.props

```diff
@@ -186,6 +186,8 @@
     <NUnitVersion>3.12.0</NUnitVersion>
     <NUnit3TestAdapterVersion>4.5.0</NUnit3TestAdapterVersion>
     <CoverletCollectorVersion>6.0.0</CoverletCollectorVersion>
+    <SystemComponentModelAnnotationsVersion>5.0.0</SystemComponentModelAnnotationsVersion>
```

Comment:
Why do we need this?

Reply:
Testing `DescriptionAttribute` resolution.